education/blockchain/mining-followup.md

```diff
@@ -0,0 +1,3 @@
+# Proof of Work
```

Comment:
I think it would be better to next another layer deeper if you want to go further.  That also helps the organization of the analogy chart when that gets built out (speaking of which the way it is displayed needs to change)